k6/load/load.js

```diff
@@ -0,0 +1,47 @@
+import http from 'k6/http';
+import {check, group, sleep, fail} from 'k6';
+
+export let options = {
+  stages: [
+    {duration: '1m', target: 3},
+    {duration: '5m', target: 6},
+    {duration: '5m', target: 9},
+    {duration: '5m', target: 15},
+    {duration: '5m', target: 9},
+    {duration: '5m', target: 6},
+    {duration: '1m', target: 2}
```

Comment:
```suggestion
    {duration: '1m', target: 19},
    {duration: '25m', target: 19},
    {duration: '1m', target: 0}
```

- 피크타임 VUser가 19로 계산하셨던데, 왜 15로 두셨을까요?
- 일정기간 동일한 트래픽을 유지하려면 위와 같이 작성하시면 되어요~